cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/12-affording-your-loan.html

```diff
@@ -186,6 +186,14 @@ <h3 class="h4 u-mt30 u-mb30">If you know your living expenses, adjust these aver
                 'data_attribute': 'data-expenses-item="item_childcare"'
             })
         }}
+       {{
+            input.render({
+                'label': 'Current debt payments',
+                'id': 'expenses__currentDebts',
+                'value': '$0',
+                'data_attribute': 'data-expenses-item="item_currentDebts"'
+            })
+        }}
```

Comment:
I'm seeing NaN as the initial value for this field when I select the monthly budget option:

<img width="692" alt="Screen Shot 2020-09-30 at 10 38 36 AM" src="https://user-images.githubusercontent.com/778171/94720394-46a8f900-0309-11eb-86c3-dd02d98d914f.png">

Reply:
Hrm. I'm having problems replicating this, but it might be due to something locally. Line ~128 in `expenses-model.js` should prevent this:

```
    if ( typeof expensesModel.values.item_currentDebts === 'undefined' ) {
      expensesModel.values.item_currentDebts = 0;
    }
```

But maybe I should initialize this and Childcare as `0` in the model itself. Actually, a refactor of expenses items such that it initialized them as `0` based on what is found in the DOM would be a good idea (that's what we do with financial items).